src/test/java/org/assertj/core/error/OptionalShouldContain_create_Test.java

```diff
@@ -71,4 +72,14 @@ public void should_create_error_message_with_optionallong() {
                                        "  <10L>\n" +
                                        "but did not.");
   }
+
+  @Test
+  public void should_create_error_message_for_different_instances() {
+    String errorMessage = shouldContainSame(Optional.of(new Integer(10)), new Integer(10)).create();
+    assertThat(errorMessage).isEqualTo("\nExpecting:\n" +
```

Comment:
even though, it is done yet done for the other assertion,we should use `String.format` and `%n` to have proper line separator when running the tests from different OS, it should look like:

```java
   assertThat(message).isEqualTo(format("[Test] %n" +
                                         "Expecting:%n" +
                                         " <[\"Yoda\"]>%n" +
                                         "to contain:%n" +
                                         " <[\"Luke\", \"Yoda\"]>%n" +
                                         "but could not find:%n" +
                                         " <[\"Luke\"]>%n"));
 ```

I won't be mad if you decide to change the other tests assertions messages :smile_cat: